### PR TITLE
Xunit xml output

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by AlexGhiondea** [comment]
LGTM -- can you squash the commits before you commit?

File: tests/runtest.cmd [hunk 1]
```diff
@@ -118,12 +119,6 @@ goto :eof
 set BUILDERRORLEVEL=%ERRORLEVEL%
 goto :eof
```
Comment (ramarag) [inline tests/runtest.cmd]: can you remove this instead so that following are no longer dead code

Comment (mmitche) [inline tests/runtest.cmd]: We call to runtests, which if it returns as failing will print a message about where to look for results.  I think removing the dead code is the right option.

Comment (ramarag) [inline tests/runtest.cmd]: That meesage actually points to the msbuild log which will only contain all possible failures , the html is sufficient for most people and is much easier to use to diagnose test failures

Comment (mmitche) [inline tests/runtest.cmd]: Updated with new info in the message after the runtest call

File: tests/runtest.cmd [hunk 2]
```diff
@@ -109,7 +110,12 @@ md %Core_Root%
 xcopy /s %__BinDir% %Core_Root%
 call :runtests 
 
-IF %BUILDERRORLEVEL% NEQ 0 echo Test Run  failed. Refer %__TestRunBuildLog% for details. && exit /b %BUILDERRORLEVEL%
+IF %BUILDERRORLEVEL% NEQ 0 ( 
+    echo Test Run failed. Refer to the following"
+    echo Msbuild log: %__TestRunBuildLog%
```
Comment (ramarag) [inline tests/runtest.cmd]: this log actually shows the console output of xunit runner and steps taken to prep the run. Not building the tests

**Comment by ramarag** [comment]
LGTM

**Pull request merged**